### PR TITLE
Migrate Stocks example to latest template

### PR DESCRIPTION
**Pull request by cbracken**

### CODE REVIEW

File: examples/stocks/README.md [hunk 1]
```diff
@@ -1,9 +0,0 @@
-## Icon
-
-Icon was created using Android Asset Studio:
-https://romannurik.github.io/AndroidAssetStudio/icons-launcher.html#foreground.type=image&foreground.space.trim=0&foreground.space.pad=0&foreColor=607d8b%2C0&crop=0&backgroundShape=square&backColor=fff%2C100&effects=none
-
-From this clipart:
-https://openclipart.org/detail/30403/tango-go-up
-Which is public domain.
-
```
Comment (Hixie) [inline examples/stocks/README.md]: we shouldn't change the icon

File: examples/stocks/ios/Flutter/Debug.xcconfig [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#include "Generated.xcconfig"
+#include "Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig"
```
Comment (xster) [inline examples/stocks/ios/Flutter/Debug.xcconfig]: this isn't checked in for instance

**Comment by cbracken** [comment]
Fixes #8062 for Stocks example.

File: examples/stocks/android/app/build.gradle [hunk 3]
```diff
@@ -24,6 +24,9 @@ android {
 
     defaultConfig {
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        
+        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
```
Comment (Hixie) [inline examples/stocks/android/app/build.gradle]: presumably remove the TODO?

**Review by xster — CHANGES_REQUESTED** [review]
Same comments as https://github.com/flutter/flutter/pull/9621#pullrequestreview-35905881

We should either check in all the artifacts and make sure the project builds without running pod install first or remove all the cocoapod references in the project

File: examples/stocks/ios/Podfile [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  end
+end
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['ENABLE_BITCODE'] = 'NO'
+    end
+  end
+end
```
Comment (Hixie) [inline examples/stocks/ios/Podfile]: this stuff really seems like it should be in some generic place?

**Comment by cbracken** [comment]
> We should either check in all the artifacts and make sure the project builds without running pod install first or remove all the cocoapod references in the project

This was a straight checkin of what was created via `flutter create`, with just minimal edits to IDs and app names. It appears that if you `flutter create` a project then `git add .`, push to a remote git server, blow everything away, and then clone it back down again, it won't build. That seems fundamentally broken.

Until we've got that fixed, I'll do what I've done with the others, which is to remove plugin support in this sample.

File: examples/stocks/ios/Podfile (hunk 4 above)
Comment (cbracken) [inline examples/stocks/ios/Podfile]: Deleted until the default created by `flutter create` is check-inable. I'll file bugs.

File: examples/stocks/ios/Flutter/Debug.xcconfig (hunk 2 above)
Comment (cbracken) [inline examples/stocks/ios/Flutter/Debug.xcconfig]: Deleted.

File: examples/stocks/android/app/build.gradle (hunk 3 above)
Comment (cbracken) [inline examples/stocks/android/app/build.gradle]: Done.

**Comment by xster** [comment]
Filed #9763

File: examples/stocks/README.md (hunk 1 above)
Comment (cbracken) [inline examples/stocks/README.md]: Updated the iOS icon to match the Android one.

**Comment by Hixie** [comment]
LGTM
...modulo xster's requests

**Review by xster — APPROVED** [review]

**Pull request merged**